sfdx-source/apex-common/main/classes/fflib_SObjectDescribe.cls

```diff
@@ -216,6 +216,19 @@ public class fflib_SObjectDescribe {
 		return getDescribe(instance.getSObjectType());
 	}
 
+	private Id getRecordTypeId(String recordTypeDeveloperName)
+	{
+		return getDescribe().getRecordTypeInfosByDeveloperName().get(recordTypeDeveloperName)?.getRecordTypeId();
+	}
+
+	public static Id getRecordTypeId(String sObjectName, String recordTypeDeveloperName){
```

Comment:
Small detail; 
I see an inconsistency in brackets placements, 
opening brackets are usually places on a new line (like line 220)

Reply:
I am changing this in the next commit now.
Thanks @wimvelzeboer !

Reply:
I have completed the change in the pushed commit.